pepper-apis/dsm-core/src/test/java/org/broadinstitute/dsm/util/TestParticipantUtil.java

```diff
@@ -85,7 +87,7 @@ public static ParticipantDto createIneligibleSharedLearningParticipant(String gu
         ParticipantDto testParticipant = createParticipant(ddpParticipantId, ddpInstanceDto.getDdpInstanceId());
         ElasticTestUtil.createParticipant(esIndex, testParticipant);
         ElasticTestUtil.addParticipantProfileFromFile(esIndex, "elastic/participantProfile.json", ddpParticipantId);
-        ElasticTestUtil.addDsmObjectToParticipantFromFile(esIndex, "elastic/participantDsm.json", ddpParticipantId, dob);
+        ElasticTestUtil.addDsmObjectToParticipantFromFile(esIndex, "elastic/participantDsm.json", ddpParticipantId, dob, null);
```

Comment:
I realize this code was not added for this PR, but seems silly to have two identical methods that vary by a couple details (createSharedLearningParticipant, createIneligibleSharedLearningParticipant). Feels like they should be combined, or at least share a common sub-method?